### ISSUE TITLE
About \Delta of Distribution Focal Loss

### ISSUE THREAD

**Issue by yuyiwings** [implus/GFocal#14]
the article says \Delta means the interval between y_i and y_{i+1}.

In glocal_loss.py
`dis_left = label.long()
    dis_right = dis_left + 1`

I consider that 1 means the interval. Because F.cross_entropy needs an **integer number** to be the true label.
I find that there has a \Delta equals to 0.5 in your ablation study.(Table 2 (c))
So what is the setting of your CE loss when \Delta equals to 0.5?
